cloudant.go

```diff
@@ -53,12 +53,12 @@ type Index struct {
 }
 
 // NewClient ...
-func NewClient(username string, password string) (*Client, error) {
-	auth := couchdb.BasicAuth(username, password)
+func NewClient(username string, apikey string, password string) (*Client, error) {
+	auth := couchdb.BasicAuth(apikey, password)
```

Comment:
The username & api CAN be the same but don't HAVE to be the same. In my case I do not want to use my Cloudant username and password in my code. I just generate an API key in cloudant for my project and use that. Hence, I need a way to create a connection with an api-key. If you do not use an apikey, then username and apikey have the same content.